deployments/monitoring/templates/cronjob.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: bucketmapper
+  namespace: monitoring
+spec:
+  schedule: "*/15 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
```

Comment:
Should add in the normal hardening stuff here (security policy, not mounting the Kubernetes token, etc.)